vertical-pod-autoscaler/enhancements/7862-cpu-startup-boost/README.md

```diff
@@ -95,24 +95,37 @@ down the CPU resources to the appropriate non-boosted value:
 
 ### API Changes
 
-The new `StartupBoost` parameter will be added to the [`ContainerResourcePolicy`](https://github.com/kubernetes/autoscaler/blob/vertical-pod-autoscaler-1.3.0/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go#L191)
-and contain the following fields:
-  * `StartupBoost.CPU.Factor`: the factor by which to multiply the initial
-  resource request and limit of the containers' targeted by the VPA object.
-  * `StartupBoost.CPU.Value`: the target value of the CPU request or limit
-  during the startup boost phase.
-  * [Optional] `StartupBoost.CPU.Duration`: if specified, it indicates for how
-  long to keep the pod boosted **after** it goes to `Ready`.
+The new `StartupBoost` parameter will be added to both:
+   * [`VerticalPodAutoscalerSpec`](https://github.com/kubernetes/autoscaler/blob/vertical-pod-autoscaler-1.3.0/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go#L75):
+   Will allow users to specify the default CPU startup boost for all containers of the pod targeted by the VPA object.
+   * [`ContainerResourcePolicy`](https://github.com/kubernetes/autoscaler/blob/vertical-pod-autoscaler-1.3.0/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go#L191):
+   Will allow users to optionally customize the startup boost behavior for individual containers.
+
+`StartupBoost` will contain the following fields:
+  * [Optional] `StartupBoost.CPU.Type` (type: `string`): A string that specifies
+  the kind of boost to apply. Supported values are:
+    * `Factor`: The `StartupBoost.CPU.Value` field will be interpreted as a
+    multiplier for the recommended CPU request. For example, a value of `2` will
+    double the CPU request.
+    * `Quantity`: The `StartupBoost.CPU.Value` field will be interpreted as an
+    absolute CPU resource quantity (e.g., `"500m"`, `"1"`) to be used as the CPU
```

Comment:
I just want to sanity check this. If Type was set to Factor, and Value was set to 500m, would that be invalid and the API reject it?